R/utils.R

```diff
@@ -3,15 +3,15 @@ naomi_write_csv <- function(...) {
 }
 
 naomi_read_csv <- function(file, ...) {
-  csv_reader(file, FALSE)(file, ..., stringsAsFactors = FALSE)
+  csv_reader(file, TRUE)(file, ...)
```

Comment:
This is actually causing a bunch of test failures in hintr - why are we switching to using readr here? Do we need this to avoid encoding issues for Rachel?